generators/app/templates/src/app/components/SearchBar/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+interface Props {
+  buttonClassName?: string;
+  className?: string;
+  formClassName?: string;
+  textButtonSearch?: string;
+  children?: React.ReactNode;
```

Comment:
This should be required as well, it would only be a form with a button otherwise